database/migrations/2020_06_20_212411_create_mods_stats_table.php

```diff
@@ -30,7 +30,7 @@ public function up()
             $table->string('json_stats', 1000);
             $table->string('icon', 1000);
             $table->string('mod_type', 1000);
-            $table->unsignedBigInteger('patch_number_index');
+            $table->unsignedBigInteger('patch_id');
```

Comment:
Unfortunately, you will have to generate new migrations to fix this. Otherwise anyone that has migrated won't get the updates.